pom.xml

```diff
@@ -319,7 +324,6 @@
                         <dependency>
                             <groupId>com.puppycrawl.tools</groupId>
                             <artifactId>checkstyle</artifactId>
-                            <version>${checkstyle.version}</version>
```

Comment:
this one doesn't necessarily need to be moved, it's part of pluginManagement already. My comment in #87 was referring to the lines https://github.com/confluentinc/common/blob/master/pom.xml#L563-L574. Since we already specify the plugin dependencies and their versions here (in pluginManagement), we should be able to remove the lines I linked to entirely.